### PR TITLE
Parameters to rcnn scripts

### PR DESCRIPTION
**Pull request by danielaRiesgo**
Adding parameters to scripts and completing README.
Also, in the way, eliminating python 2 code.

### Scripts missing
`rnn_eval.py`, which is for classifying.

### CODE REVIEW

File: rcnn/build_labels.py [hunk 1]
```diff
@@ -57,7 +61,103 @@ def get_label(timestamp, classes):
...
+        create_dir_if_not_exists(FLAGS.copy_dir)
+        classes_per_video = [class_per_frame[video].keys() for video in batches]
+        for i in range(len(classes_per_video)):
+            for klass in classes_per_video[i]:
+                class_path = os.path.join(FLAGS.copy_dir, klass)
+                video_path = os.path.join(class_path, batches[i])
+                create_dir_if_not_exists(class_path)
+                create_dir_if_not_exists(video_path)
+
+
```
Comment (fmesteban) [inline rcnn/build_labels.py]: extra empty line

File: rcnn/build_labels.py [hunk 2]
```diff
@@ -57,7 +61,103 @@ def get_label(timestamp, classes):
...
+    parser.add_argument(
+        '--videos_dir',
+        type=str,
+        default='/tmp/videos',
+        help="""\
+        Where to find the recorded videos.\
+        """
+    )
+    parser.add_argument(
+		'--classes_file',
```
Comment (fmesteban) [inline rcnn/build_labels.py:103]: extra indent?

File: rcnn/build_labels.py [hunk 3]
```diff
@@ -57,7 +61,103 @@ def get_label(timestamp, classes):
...
+		help="""\
+		Name of the file (without extension) where the classes
+        for each video are specified.
+        This file must be in the same folder as this file,
+        and it must define a dictionary with the classes
+        in the correct format (check README).\
+		"""
+	)
+    parser.add_argument(
+		'--classes_dict',
```
Comment (fmesteban) [inline rcnn/build_labels.py:115]: same

File: rcnn/build_labels.py (hunk 2 above)
Comment (danielaRiesgo) [reply rcnn/build_labels.py:103]: Nop, looks good in sublime

File: rcnn/build_labels.py (hunk 3 above)
Comment (danielaRiesgo) [reply rcnn/build_labels.py:115]: Same here

**Review by fmesteban — DISMISSED** [review]

**Review by fmesteban — APPROVED** [review]

**Pull request merged**